docs/environment_variables.md

```diff
@@ -2,6 +2,13 @@ Environment Variables
 =====================
 
 The HTTPX library can be configured via environment variables.
+To activate the environment variables, `trust_env` has to be set `true`.
```

Comment:
We should use `True` to match Python's sentinel

Reply:
With environment variables I guess we want to treat them as case insensitive.

I'd probably provide examples as `"true"`, in order to make it more visually distinct to the Python `True` keyword/singleton instance.

Reply:
But this is a Python instance of True?